camp_fin/models.py

```diff
@@ -1065,68 +1018,43 @@ def stack_trends(trend):
         # Donations and expenditures
         monthly_query = """
             SELECT
-              {table}.amount AS amount,
-              {table}.month AS month
-            FROM {table}_by_month AS {table}
-            WHERE {table}.entity_id = %s
-              AND {table}.month >= '{year}-01-01'::date
-            ORDER BY month
+              months.year,
+              months.month,
+              {table}.amount
+            FROM (
+              SELECT
```

Comment:
optional suggestion: this looks like a good place to use a CTE.

Reply:
CTEs aren't supported in the version of Postgres we're using for local development (which might not be a constraint in deployment, actually, but don't feel like upgrading Postgres here).